src/System.Collections/src/System/Collections/Generic/SortedSet.TreeSubSet.cs

```diff
@@ -45,12 +45,6 @@ public TreeSubSet(SortedSet<T> Underlying, T Min, T Max, bool lowerBoundActive,
                 VersionCheckImpl();
             }
 
-            private TreeSubSet(SerializationInfo info, StreamingContext context)
```

Comment:
Hmm.  TreeSubSet is a type derived from SortedSet and is returned from methods on SortedSet like GetViewBetween.  As far as the consumer is concerned, it's a SortedSet... seems odd that some SortedSets from the framework could be serialized and others couldn't, depending on from which method you got them.  Maybe it should be on the list?